frontend/src/components/general/Ranking/Ranking.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Table } from 'react-bootstrap'
+import { getHeroName } from '../../../utils/constants'
+import { TableBodyRow } from '../../professor/GameManagement/TableStyles'
+import { TableContainer } from './RankingStyle'
+
+function Ranking(props) {
+  return (
+    <TableContainer>
+      <Table className={'my-0'} style={{ color: 'var(--font-color)' }}>
+        <thead>
+          <tr>
+            <th>Nazwa gracza</th>
```

Comment:
Może lepiej po prostu "Gracz"?
```suggestion
            <th>Gracz</th>
```